src/model/Request.php

```diff
@@ -5,18 +5,45 @@
 // hnusna skareda staticka vec! fuj!
 class Request
 {
-    public static function getIp()
+    public static function getIp(): string
     {
-        if (!empty($_SERVER['HTTP_CLIENT_IP'])) {
-            $ip = $_SERVER['HTTP_CLIENT_IP'];
-        } elseif (!empty($_SERVER['HTTP_X_FORWARDED_FOR']) && $_SERVER['HTTP_X_FORWARDED_FOR'] != '127.0.0.1') {
-            $ip = $_SERVER['HTTP_X_FORWARDED_FOR'];
-        } elseif (!empty($_SERVER['REMOTE_ADDR'])) {
-            $ip = $_SERVER['REMOTE_ADDR'];
-        } else {
-            $ip = 'cli';
+        $serverKeysToCheck = [
```

Comment:
The order of server keys changed comparing the original implementation. Was there a reason for it?  
(it's possibly a breaking change)

Reply:
I can not really prove, that the changed order is better (as far as I know, none of these two headers are standardised), but from my (our) experience, the HTTP_X_FORWARDED_FOR header is more widely used, we could get better, more relevant results using this as the first one to check..